content/blog/pulumi-kubernetes-operator-1-0/index.md

```diff
@@ -6,6 +6,9 @@ tags: ["Kubernetes", "Continuous-Delivery", "operators"]
 meta_desc: "Pulumi Kubernetes Operator 1.0: GitOps, Automation API, State Backends, Safe Upgrades and Metrics"
 meta_image: operator.png
 ---
+{{% notes type="info" %}}
+Check out the version 2.0 beta of the [Pulumi Kubernetes Operator](/blog/pulumi-kubernetes-operator-2-0/).
```

Comment:
remove the beta because we probably won't touch it again after it goes out of beta